src/topp/GNPSExport.cpp

```diff
@@ -112,9 +115,15 @@ class TOPPGNPSExport : public TOPPBase
     registerInputFileList_("in_mzml", "<files>", ListUtils::create<String>(""), "Original mzml files containing the ms2 spectra (aka peptide annotation). \nMust be in order that the consensusXML file maps the original mzML files.");
     setValidFormats_("in_mzml", ListUtils::create<String>("mzML"));
 
-    registerOutputFile_("out", "<file>", "", "Output MGF file");
+    registerOutputFile_("out", "<file>", "", "Output MGF file.");
     setValidFormats_("out", ListUtils::create<String>("mgf"));
 
+    registerOutputFile_("out_quantification", "<file>", "", "Output feature quantification table.");
+    setValidFormats_("out_quantification", ListUtils::create<String>("txt"));
```

Comment:
use `{"txt"}` for bevity and speed :)

Reply:
What do you mean exactly? Does this apply to the other formats as well?

Reply:
I meant:
`setValidFormats_("out_quantification", {"txt"});`

> Does this apply to the other formats as well?
Yes. If you find some others nearby, feel free to fix :)